scripts/file_resume.js

```diff
@@ -23,6 +23,42 @@ window.addEventListener("DOMContentLoaded", function() {
         return true;
     }
 
+    // polyfill for file.arrayBuffer()
+    function fileToArrayBuffer(file) {
```

Comment:
same here, do we need a polyfil? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer

Reply:
It's File:arrayBuffer()  we need which seems to be same as Blob: arrayBuffer() which is not supported by any of the browsers we can use.